src/lib/components/cards/cardWithTopIcon/index.tsx

```diff
@@ -29,7 +31,12 @@ export default ({
       } ${className ?? ''}`}
       {...props}
     >
-      <img width="48xp" height="48px" alt={topIcon.alt} src={topIcon.src} />
+      <img
+        width={topIconSize ? `${topIconSize?.width}px` : '48xp'}
```

Comment:
It'll be better if you set `topIconSize` with a default value of `48px`, then it'll save you this ternary operator.